packages/wdio-webdriver-mock-service/src/WebDriverMock.ts

```diff
@@ -22,7 +24,10 @@ for (const protocol of protocols) {
 }
 
 export default class WebDriverMock {
-    constructor(host = 'localhost', port = 4444, path = '/') {
+    command: ProxyConstructor
+    path: string
+    scope: any
```

Comment:
this should be a nock instance, no?

Reply:
yes, but if i changes the type to nock.Scope , then i am not able to dynamically invoked the method at this line
https://github.com/webdriverio/webdriverio/pull/5968/files/83ef658b8e988b71c01ed38834275a98bdf7d4a5#diff-93b5d914eb7c597cb96b94d355ad46af9e6a31bdc4fc8fb0d4d9a6dc6803c120R75

Reply:
What does it say? We can cast `method` to `keysof nock.Scope` or something like that.

Reply:
Below is the error message i am getting:

`Element implicitly has an 'any' type because type 'Scope' has no index signature. Did you mean to call 'get'?ts(7052)`

![image](https://user-images.githubusercontent.com/45997477/96726843-07ce0680-13d0-11eb-90fc-d06be49a7f3c.png)